Bottom: e26160a4d7aa7ebd8e7e0183cf0a8185df7cef15
Top:    cf5bf1afd5864703dd6a49a86c3b667b9519f796
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-01-02 21:26:53 -0500

Refresh of zarlink

---

diff --git a/package/kernel/linux/modules/sound.mk b/package/kernel/linux/modules/sound.mk
index f7e3e7d..5d3d62a 100644
--- a/package/kernel/linux/modules/sound.mk
+++ b/package/kernel/linux/modules/sound.mk
@@ -169,6 +169,7 @@ define KernelPackage/sound-soc-core
 	CONFIG_SND_SOC_DMAENGINE_PCM=y \
 	CONFIG_SND_SOC_ALL_CODECS=n
   FILES:=$(LINUX_DIR)/sound/soc/snd-soc-core.ko
+  DEPENDS:=+kmod-sound-core
   AUTOLOAD:=$(call AutoLoad,55, snd-soc-core)
   $(call AddDepends/sound)
 endef
